src/workloads/sharding/WriteOneWithoutShardKeyShardedCollection.yml

```diff
@@ -0,0 +1,127 @@
+# TODO SERVER-3208: Enable Workload.
```

Comment:
If you want to also include the sharded workload we might want to change the ticket description, and file a PERF ticket to enable the sharded workload later.